### PR TITLE
Coded design for ContributorPage

### PR DESCRIPTION
**Pull request by JasonLin43212**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ const Flags =
     },
     error_sound       :
     {
-        defaultValue : false,
+        defaultValue : true,
```
Comment (jkao1) [inline gulpfile.js]: What does this do?

File: src/js/modules/users/components/ContributorPage.js [hunk 2]
```diff
@@ -1,49 +1,85 @@
 import React from "react";
 import { connect } from "react-redux";
-import { Link } from "react-router-dom";
 import injectSheet from "react-jss";
+import { Grid, Row, Col } from "react-bootstrap/lib";
 
 import { getContributorFromSlug } from "../selectors";
 import { getArticlesByContributor } from "../../articles/selectors";
 import { getSections } from "../../sections/selectors";
+import { ArticleRow } from "../../articles/components";
```
Comment (jkao1) [inline src/js/modules/users/components/ContributorPage.js]: Why are you importing an ArticleRow? Just use ArticleList. ArticleLists are containers for ArticleRow, and all you have to do is feed the ArticleList its props (listed in ArticleList.js).

File: gulpfile.js (hunk 1 above)
Comment (JasonLin43212) [inline gulpfile.js]: oh shoot, i should change that back. I thought it was suppose to make some kind of sound if there was an error, but it didn't seem to work so i should change it back.

File: src/js/modules/users/components/ContributorPage.js (hunk 2 above)
Comment (JasonLin43212) [inline src/js/modules/users/components/ContributorPage.js]: `ArticleList` takes in the prop `section`. However, a user can write articles for multiple sections so by using `ArticleRow`, I can specify which section belongs to which article.

Comment (jkao1) [inline src/js/modules/users/components/ContributorPage.js]: got it

**Pull request closed without merging**